less/mixins.less

```diff
@@ -0,0 +1,46 @@
+@import 'variables.less';
+
+.flex(@direction, @justify, @align) {
+  display: flex;
+  flex-direction: @direction;
+  justify-content: @justify;
+  align-items: @align;
+}
+
+.border {
+  border: 1px solid black;
+}
+
+.border-radius(@radius) {
+  border-radius: @radius;
+}
+
+.header(@fontSize) {
+  font-weight: bold;
+  font-size: @fontSize;
+  font-family: @fontSectionHeader;
+  margin: 1rem 0;
+
+  @media (max-width: @mobileViewBreakpoint) {
+    text-align: center;
+  }
+}
```

Comment:
Very well done with all these mixins you wrote. 
I guess it can take up time to write these, and for these small projects could feel pointless. But as a project grows these are going to speed up your work so much!